Loenn/triggers/momentummodifier.lua

```diff
@@ -0,0 +1,85 @@
+local momentum = {}
+
+momentum.name = "ContortHelper/MomentumModifierTrigger"
+momentum.placements = {
+    name = "default",
+    data = {
+        modifierX = 0.0,
+        modifierY = 0.0,
+        operation = "Set",
+        onlyX = false,
+        onlyY = false,
+        neededFlags = "",
+        flagsAfterInvoke = "",
+        delay = 0.0,
+        occurOnEnter = true,
+        oneUse = false,
+        persistent = false
+    }
+}
+
+local conditionalMomentum = {}
+
+conditionalMomentum.name = "ContortHelper/ConditionalMomentumModifierTrigger"
+conditionalMomentum.placements = {
+    name = "default",
+    data = {
+        modifierX = 0.0,
+        modifierY = 0.0,
+        conditionalModifierX = 0.0,
+        conditionalModifierY = 0.0,
+        operation = "Set",
+        conditionalOperation = "EqualTo",
+        onlyX = false,
+        onlyY = false,
+        conditionalOnlyX = false,
+        conditionalOnlyY = false,
+        neededFlags = "",
+        flagsAfterInvoke = "",
+        delay = 0.0,
+        occurOnEnter = true,
+        oneUse = false,
+        persistent = false
+    }
+}
+
+momentum.fieldInformation = {
+    operation = {
+        options = {
+            "Set",
+            "Addition",
+            "Subtraction",
+            "Multiplication",
+            "Division"
+        },
+        editable = false
+    }
+}
+
+conditionalMomentum.fieldInformation = {
+    operation = {
+        options = {
+            "Set",
+            "Addition",
+            "Subtraction",
+            "Multiplication",
+            "Division"
+        },
+        editable = false
+    },
+    conditionalOperation = {
+        options = {
+            "LessThan",
+            "LessThanOrEqualTo",
+            "EqualTo",
+            "GreaterThanOrEqualTo",
+            "GreaterThan"
+        },
+        editable = false
+    }
+}

```

Comment:
For operation option, use the tables defined the in the `libraries/contorthelper.lua` library, namely, `contorthelper.operations` and `contorthelper.conditionals`
```suggestion
-- move this require call to the top of the file.
local contorthelper = require("mods").requireFromPlugin("libraries.contorthelper")

momentum.fieldInformation = {
    operation = {
        options = contorthelper.operations,
        editable = false
    }
}

conditionalMomentum.fieldInformation = {
    operation = {
        options = contorthelper.operations,
        editable = false
    },
    conditionalOperation = {
        options = contorthelper.conditionals,
        editable = false
    }
}
```